### DISCUSSION TITLE
Favicon with github pages

### DISCUSSION THREAD

**Discussion by Vic-ST** [community/community#22625] [Other Feature Feedback, Questions, & Ideas]
<p>How do you create a favicon for github pages? Is there a website that you do it with?</p>

**Comment by roryokane** [comment]
<p>Instead of uploading a file to /favicon.ico, you specify the favicon inside the &lt;head&gt; of each HTML page, like this:</p>
<pre><code>&lt;link rel="icon" href="/YourIcon.ico" type="image/x-icon" /&gt;
</code></pre>

**Comment by Vic-ST** [comment]
<p>Okay let me try that.</p>

**Comment by ghost** [comment]
<p>On top of what you said, <a class="mention" href="/roryokane">@roryokane</a>, if you have the <strong>_includes/head.html</strong>  file in your repository or a <strong>default.html</strong> depending on the theme, you can edit the  <strong>&lt;head&gt;</strong>  section to include the favicon:</p>
<pre><code>&lt;head&gt;
&lt;!-- Default head tags --&gt;
  &lt;meta charset="utf-8"&gt;
  &lt;meta http-equiv="X-UA-Compatible" content="IE=edge"&gt;
  &lt;meta name="viewport" content="width=device-width, initial-scale=1"&gt;
  &lt;link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}"&gt;
  &lt;link rel="alternate" type="application/rss+xml" title="{{ site.title | escape }}" href="{{ "/feed.xml" | relative_url }}"&gt;

&lt;!-- Favicon head tag --&gt;
&lt;link rel="icon" href="PATH/TO/favicon.ico" type="image/x-icon"&gt;
&lt;/head&gt;
</code></pre>
<p>(I’m using the <em>minima</em> theme for my site, if you’re wondering)</p>
<p>It really depends what theme you’re using :slight_smile:</p>

**Comment by birdcar** [comment]
<p>Were you able to find a solution here? And just to clarify, are you asking how to <em>create</em> a Favicon? Or just how to include one in your HTML?</p>

**Comment by Vic-ST** [comment]
<p>How to create one. And no, I have not found an answer</p>

**Comment by Siedlerchr** [comment, marked as answer]
<p>You need to save an image in several formats, e.g. 32x32 or use this tool here:</p>
<p><a href="https://realfavicongenerator.net/" rel="nofollow noopener">https://realfavicongenerator.net/</a></p>

**Comment by Vic-ST** [comment]
<p>Thanks man!</p>

**Comment by jhvanderschee** [comment]
<p>Well… for 99% procent of the cases you need only one format these days: a 196x196 pixel PNG with transparency.</p>
<p>Just link it in your head like this:</p>
<pre><code>&lt;link rel="shortcut icon" type="image/png" href="/img/icon-196x196.png"&gt;
  &lt;link rel="shortcut icon" sizes="196x196" href="/img/icon-196x196.png"&gt;
  &lt;link rel="apple-touch-icon" href="/img/icon-196x196.png"&gt;
</code></pre>
<p>I got the code from <a href="http://jekyllcodex.org/without-plugin/seo/" rel="nofollow noopener">http://jekyllcodex.org/without-plugin/seo/</a>.</p>

**Comment by JonDevOps** [comment]
<p>Put this in your web pages head:</p>
<pre><code>&lt;link rel="shortcut icon" type="image/x-icon" href="favicon.ico"&gt;
</code></pre>

**Comment by sorianog** [comment]
<p><a class="mention" href="/timrossback">@timrossback</a> Exactly what I needed ( <strong>_includes/head.html</strong> ). Thanks! :smile:</p>
<p>  <img src="https://user-images.githubusercontent.com/13261711/181072213-59ac68f3-aedc-4e47-ad0b-0f0e86ccd1e6.png" alt="Screen Shot 2018-10-10 at 3.42.13 AM.png" data-base62-sha1="eepMD6inXqSTklpWdVrkvmswQmD" width="404" height="152"></p>

**Comment by Vladimir-Moskalenko** [comment]
Is there a way to add an already created icon to a blank site? Without a theme?
